README.md

```diff
@@ -100,5 +130,23 @@ Once in Vanilla JS, and once in jQuery, write a function that adds an event list
 
 Your Answer:
 ```js
+
+<button class="submit-quiz"></button>
+
+submit-quiz.addEventlistener("click" , function(){
```

Comment:
Where did you define the var `submit-quiz`?

You could select that button in save it to a var in vanilla like:
```js
var submit-quiz = document.body.querySelector(".submit-quiz")
```